index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Home</title>
+  </head>
+  <body>
+   <div class="wrapper">
+     <header class="toolbar">
+       <div class="logo">
+         <img src="./image/images-head/logo.png" alt="MyLogo" />
+       </div>
+       <nav class="nav-btn">
+         <img src="./image/images-head/union.png" alt="navbarimage" />
```

Comment:
- [ ] Please align your hamburger with the logo, check your design:
![image](https://user-images.githubusercontent.com/60631456/121391861-7090d180-c914-11eb-9cef-341060334f43.png)
 and the original:
![image](https://user-images.githubusercontent.com/60631456/121391936-87cfbf00-c914-11eb-8bd7-996f1f214719.png)

Reply:
Hamburger aligned to the logo as requested.